service/src/main/java/uk/gov/hmcts/reform/fpl/service/InboxLookupService.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.config.DefaultEmailLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityEmailLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.Solicitor;
+
+import java.util.Optional;
+
+import static org.apache.commons.lang.StringUtils.isNotBlank;
+
+@Service
+public class InboxLookupService {
+    private final ObjectMapper objectMapper;
+    private final LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration;
+    private final DefaultEmailLookupConfiguration defaultEmailLookupConfiguration;
+
+    public InboxLookupService(ObjectMapper objectMapper,
```

Comment:
Do you not need to `@Autowire` your constructor?

Reply:
added @Autowired